src/main/java/com/TableTalk/Enterprise/controllers/TableTalkController.java

```diff
@@ -1,168 +1,119 @@
 package com.TableTalk.Enterprise.controllers;
 
+import com.TableTalk.Enterprise.dto.Game;
+import com.TableTalk.Enterprise.dto.ProfilePicture;
+import com.TableTalk.Enterprise.dto.Room;
+import com.TableTalk.Enterprise.dto.User;
 import com.TableTalk.Enterprise.services.ITableTalkService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpHeaders;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
-import com.TableTalk.Enterprise.dto.Game;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.*;
+
 import java.io.IOException;
 import java.util.List;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.*;
 
 
 @Controller
-    public class TableTalkController {
-
+public class TableTalkController {
 
     @Autowired
     ITableTalkService TableTalkService;
-        /**
-         * Handle the root (/) endpoint and return a start page.
-         * @return
-         */
```

Comment:
Having JavaDoc doesn't make sense if it's not "touching" the method its explaining.